### ISSUE TITLE
warnings during docker build

### ISSUE THREAD

**Issue by jazzl0ver** [Awesome-Technologies/synapse-admin#3]
```
# docker build -t synapse-admin:latest .
Sending build context to Docker daemon  964.6kB
Step 1/8 : FROM node:10-alpine as builder
10-alpine: Pulling from library/node
e7c96db7181b: Pull complete
50958466d97a: Pull complete
56174ae7ed1d: Pull complete
284842a36c0d: Pull complete
Digest: sha256:77c898d0da5e7bfb6e05c9a64de136ba4e03889a72f3c298e95df822a38f450d
Status: Downloaded newer image for node:10-alpine
 ---> b95baba1cfdb
Step 2/8 : WORKDIR /src
 ---> Running in 0358c9d72861
Removing intermediate container 0358c9d72861
 ---> efc67a523905
Step 3/8 : COPY . /src
 ---> 333b4bc993d3
Step 4/8 : RUN yarn --network-timeout=100000 install
 ---> Running in db3a8b2aa80e
yarn install v1.17.3
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
warning babel-preset-react-app > babel-preset-env > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning extract-text-webpack-plugin@3.0.2: Deprecated. Please use https://github.com/webpack-contrib/mini-css-extract-plugin
warning jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()
warning react-router-redux@5.0.0-alpha.9: This project is no longer maintained.
warning rekit-core > trash > xdg-trashdir > @sindresorhus/df > execa > cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
warning sw-precache-webpack-plugin > sw-precache@5.2.1: Please migrate to Workbox: https://developers.google.com/web/tools/workbox/guides/migrations/migrate-from-sw
warning sw-precache-webpack-plugin > sw-precache > sw-toolbox@3.6.0: Please migrate to Workbox: https://developers.google.com/web/tools/workbox/guides/migrations/migrate-from-sw
[2/4] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > babel-jest@24.9.0" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-jest > babel-preset-jest@24.9.0" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-jest > babel-preset-jest > @babel/plugin-syntax-object-rest-spread@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > css-loader@3.2.0" has incorrect peer dependency "webpack@^4.0.0".
warning "rekit-core > eslint-config-airbnb-base@12.1.0" has incorrect peer dependency "eslint@^4.9.0".
warning " > sass-loader@6.0.7" has unmet peer dependency "node-sass@^4.0.0".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 86.33s.
...
```

Would also be excellent if you published the image on docker hub!

**Comment by awesome-manuel** [comment]
The warnings appear since the Docker image is build using yarn, but the project itself is not migrated yet.
https://hub.docker.com/r/awesometechnologies/synapse-admin

**Comment by jazzl0ver** [comment]
Would you please put the hub link into the README.md?

**Comment by awesome-manuel** [comment]
I'd rather put it in the project description.

**Comment by awesome-manuel** [comment]
Some warnings are resolved with 62b94b5b0515e02e8fc85e6c772bb77249ece420
